lightningd/channel/channel.c

```diff
@@ -32,9 +32,10 @@
 #include <wire/wire_io.h>
 #include <wire/wire_sync.h>
 
-/* stdin == requests, 3 == peer */
+/* stdin == requests, 3 == peer, 4 = gossip */
 #define REQ_FD STDIN_FILENO
 #define PEER_FD 3
+#define GOSSIP_FD 4
```

Comment:
Interesting, I did not know that we could determine the FD without reading the message first. I take it that actually passing the fd over the connection will map the other fd onto this processes' fd space in this deterministic manner?